x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_enrichment/processors/grok/grok_patterns_editor.tsx

```diff
@@ -157,6 +158,7 @@ const DraggablePatternInput = ({
   onChange,
   onRemove,
 }: DraggablePatternInputProps) => {
+  const resizeCheckerSetup = useResizeCheckerUtils();
```

Comment:
Can you push this down into the Expression component? It doesn't seem to be needed for anything at this hierarchy level